### ISSUE TITLE
Auto-hide relay locations that haven't checked in in a while

### ISSUE THREAD

**Issue by pmeenan** [WPO-Foundation/webpagetest#33]
WPT has a great feature that automatically hides devices that haven't connected in a while, to keep them from accumulating a large backlog and frustrate users. 
Relayed locations don't benefit from that, and will show as live even if they've been dead for a while. 

The auto-hide functionality should be expanded to include relayed locations.
On a similar note, the "time since last check" for relayed locations should also be available through the checklocations.php and other scripts to allow users of the API identify a location is inactive too. 

Implementation note: Periodically in the getwork.php cron support we should connect to all remote locations and fetch the "last check" information and populate the local .tm files.  Everything else should automatically work once that is in place.  

Update frequency does not need to be more than every 15 minutes since the threshold is 30-60 minutes but it wouldn't be a bad idea to do it a little more frequently (say every 5 minutes).  We could potentially grab the backlog at the same time so we can provide some feedback on overloaded locations.

https://code.google.com/p/webpagetest/issues/detail?id=75

**Comment by tkadlec** [comment]
We're in the process of cleaning up issues on this project in order to ensure we're able to stay on top of high priority bugs and feature requests. 

As a part of this process, we're automatically closing any issues that have had no activity within the last two years, including this one, since the codebase has changed dramatically in that time.

If you feel this is still relevant, please [file a new issue](https://github.com/WPO-Foundation/webpagetest/issues/new/choose) using the relevant issue template so we can get it prioritized. 

Thanks!
